src/test/utils/getMembers.test.js

```diff
@@ -0,0 +1,131 @@
+import {
+  getEveryMember,
+  getEveryMemberOfHousehold,
+  isHeadOrSpouse,
+  isNotHeadOrSpouse,
+  getHeadOrSpouseMembers,
+  getHeadOrSpouseOfHousehold,
+  isDependent,
+  getDependentMembers,
+  getDependentsOfHousehold,
+  isDisabled,
+  getDisabledMembers,
+  getDisabledOfHousehold,
+  isUnder13,
+  getUnder13Members,
+  getUnder13OfHousehold
+} from '../../utils/getMembers';
+
+// getEveryMember(array, fn) = array.filter(fn)
+test('getEveryMember()', () => {
+  expect(getEveryMember([0, 1, 0], val => val === 0)).toHaveLength(2);
+});
+
+// getEveryMemberOfHousehold(obj, fn) = obj.household.filter(fn)
+test('getEveryMemberOfHousehold()', () => {
+  const client = { household: [0, 1, 0] };
+  expect(getEveryMemberOfHousehold(client, val => val === 0)).toHaveLength(2);
+});
+
+describe('head or spouse', () => {
+  const headOrSpouse = ['head', 'spouse'].map(role => ({ m_role: role }));
+  const notHeadOrSpouse = ['Head', 'member', ''].map(role => ({
+    m_role: role
+  }));
+  const allMembers = headOrSpouse.concat(notHeadOrSpouse);
+
+  test('isHeadOrSpouse()', () => {
+    headOrSpouse.forEach(member => expect(isHeadOrSpouse(member)).toBe(true));
+    notHeadOrSpouse.forEach(member =>
+      expect(isHeadOrSpouse(member)).toBe(false)
+    );
+  });
+
+  // !isHeadOrSpouse()
+  test('isNotHeadOrSpouse()', () => {
+    headOrSpouse.forEach(member =>
+      expect(isNotHeadOrSpouse(member)).toBe(false)
+    );
+    notHeadOrSpouse.forEach(member =>
+      expect(isNotHeadOrSpouse(member)).toBe(true)
+    );
+  });
+
+  test('getHeadOrSpouseMembers()', () => {
+    expect(getHeadOrSpouseMembers(allMembers)).toHaveLength(2);
+  });
+
+  test('getHeadOrSpouseOfHousehold()', () => {
+    const client = { household: allMembers };
+    expect(getHeadOrSpouseOfHousehold(client)).toHaveLength(2);
+  });
+});
+
+describe('dependents', () => {
+  const dependents = [
+    { m_age: 18, m_role: 'member' },
+    { m_age: 17, m_role: 'member' },
+    { m_age: 19, m_disabled: true, m_role: 'member' }
+  ];
+  const notDependents = [
+    { m_age: 18, m_role: 'head' },
+    { m_age: 18, m_role: 'spouse' },
```

Comment:
Next iteration, might check disabled head and spouse too.

Reply:
Added them.